src/AppBundle/Committee/CommitteeManager.php

```diff
@@ -2,59 +2,85 @@
 
-    public function findCommitteeHostsList(Committee $committee): AdherentCollection
+    public function getCommitteeHosts(Committee $committee): AdherentCollection
     {
-        $uuids = $this
-            ->membershipRepository
-            ->findHostMemberships((string) $committee->getUuid())
-            ->getAdherentUuids();
-
-        return $this->findAdherentsList($uuids);
+        return $this->registry
+            ->getRepository(CommitteeMembership::class)
+            ->findHostMembers($committee->getUuid()->toString());
     }
 
-    public function findCommitteeFollowersList(Committee $committee, bool $withHosts = self::INCLUDE_HOSTS): AdherentCollection
+    public function getCommitteeFollowers(Committee $committee, bool $withHosts = self::INCLUDE_HOSTS): AdherentCollection
     {
-        $uuids = $this
-            ->membershipRepository
-            ->findFollowerMemberships((string) $committee->getUuid(), $withHosts)
-            ->getAdherentUuids();
+        return $this->registry
+            ->getRepository(CommitteeMembership::class)
+            ->findFollowers($committee->getUuid()->toString(), $withHosts);
+    }
 
-        return $this->findAdherentsList($uuids);
+    public function getOptinCommitteeFollowers(Committee $committee, bool $withHosts = self::INCLUDE_HOSTS): AdherentCollection
+    {
+        return $this->getCommitteeFollowers($committee, $withHosts)->getCommitteesNotificationsSubscribers();
     }
 
-    public function findOptinCommitteeFollowersList(Committee $committee, bool $withHosts = self::INCLUDE_HOSTS): AdherentCollection
+    public function getNearbyCommittees(Coordinates $coordinates, $limit = self::COMMITTEE_PROPOSALS_COUNT)
     {
-        return $this->findCommitteeFollowersList($committee, $withHosts)->getCommitteesNotificationsSubscribers();
+        $data = [];
+        $committeeMembershipRepository = $this->registry->getRepository(CommitteeMembership::class);
+        $committees = $this->registry->getRepository(Committee::class)->findNearbyCommittees($limit, $coordinates);
+
+        foreach ($committees as $committee) {
+            $uuid = $committee->getUuid()->toString();
+
+            $data[$uuid] = [
+                'committee' => $committee,
+                'memberships_count' => $committeeMembershipRepository->countMembers($uuid),
+            ];
+        }
+
+        return $data;
     }
 
-    private function findAdherentsList(array $uuids): AdherentCollection
+    /**
+     * @param Adherent $adherent   The follower
+     * @param string[] $committees An array of committee uuids
+     */
+    public function followCommittees(Adherent $adherent, array $committees)
     {
-        return $this->adherentRepository->findList($uuids);
+        if (empty($committees)) {
+            return;
+        }
+
+        $manager = $this->registry->getManager();
+
+        foreach ($this->registry->getRepository(Committee::class)->findByUuid($committees) as $committee) {
```

Comment:
The `findByUuid` method now exists in the `CommitteeRepository` class in my PR #205 . Can you make a `findByUuids()` method instead in the `CommitteeRepository` class please?